cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8080",
```

Comment:
if `npm run serve` has to use a port above `8080` (bc. `8080` is already used), this fails... is it possible to somehow link cypress to the test server to automatically find the correct port?

Reply:
we could force the port to `8080` when we launch the server for testing, and fail if the port is already used

Reply:
so a fully 'auto-port-selection' for both serving and testing is not possible?